src/modules/ie-objects/ie-objects.consts.tsx

```diff
@@ -377,8 +378,54 @@ export const MEDIA_ACTIONS = ({
  * Metadata
  */
 
+const getOcrMetadataFields = (simplifiedAlto: SimplifiedAlto | null): MetadataItem[] => {
+	const metadataFields: MetadataItem[] = [];
+	if (simplifiedAlto) {
+		if (simplifiedAlto.description.softwareName) {
```

Comment:
Isn't it better to use a switch case in this situation?

Reply:
no cause each case should be executed. If i only wanted to add one a switch would be the better option